tests/wakeup.rs

```diff
@@ -10,13 +10,13 @@ use tokio::reactor::Reactor;
 fn works() {
     let mut r = Reactor::new().unwrap();
     r.handle().wakeup();
-    r.turn(None);
+    let _ = r.turn(None);
```

Comment:
Could these all turn into `unwrap` as they're not supposed to fail?

Reply:
Done

Reply:
This isn't yet changed to unwrap (for me on Github at least).